client/client-multi/client-c/src/kaa/platform-impl/common/kaa_tcp_channel.c

```diff
@@ -554,26 +555,33 @@ kaa_error_t kaa_tcp_channel_set_access_point(void *context, kaa_access_point_t *
     /* In case if  channel */
     if (*channel->supported_services != KAA_EXTENSION_BOOTSTRAP) {
         error = ext_get_signature(channel->encryption.aes_session_key, channel->encryption.aes_session_key_size,
-                                  &channel->encryption.signature, &channel->encryption.signature_size);
+                &channel->encryption.signature, &channel->encryption.signature_size);
         if (error) {
-          KAA_LOG_ERROR(channel->logger, error, "Can't get signature");
+            KAA_LOG_ERROR(channel->logger, error, "Can't get signature");
         }
     }
-#endif
+
     return error;
+#endif
 
-error_insufficient_buffer:
-    KAA_LOG_ERROR(channel->logger, KAA_ERR_INSUFFICIENT_BUFFER, "Kaa TCP channel new access point [0x%08X], "
-                  "insufficient connection data length  %zu, position %zu", channel->access_point.id,
-                  connection_data_len, (position + remaining_to_read));
+cleanup:
+    if(error) {
```

Comment:
1. CS
2. No need to check if error occurred. `cleanup` mark is  (at least it _must be_) reached only if error occur.